arangod/IResearch/IResearchDocument.h

```diff
@@ -117,11 +124,27 @@ struct Field {
     return true;
   }
 
-  irs::flags const* _features{&irs::flags::empty_instance()};
+  void setFeatures(AnalyzerPool::AnalyzerFeatures const& poolFeatures) {
+    _features = &poolFeatures;
+    //FIXME: check here datastore version and translate accordingly
+    for (auto const& feature : _features->field_features()) {
```

Comment:
I suggest we do it at AnalyzerPool creation, potenitally we can have 2 feature sets, e.g. old and a new one